docs/_layouts/default.html

```diff
@@ -4,19 +4,11 @@
         <title>{{ page.title }}</title>
         <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
-        {% if jekyll.environment == "development" %}
         {% if page.ds != 4 %}
         <link type="text/css" rel="stylesheet" href="{{ page.static_dir }}/skin-full.min.css"/>
```

Comment:
Looks great! One thing: don't we need to change this static directory to point to the local?

Reply:
It should be. The static directory is just `static` which will be relative to this directory.